docs/catalyst_cites.bib

```diff
@@ -1,3 +1,44 @@
+@article{CicalaAea2025,
+Author = {Cicala, Steve},
+Title = {Restructuring the Rate Base},
+Journal = {AEA Papers and Proceedings},
+Volume = {115},
+Year = {2025},
+Month = may,
+Pages = {363–68},
+DOI = {10.1257/pandp.20251112},
+URL = {https://www.aeaweb.org/articles?id=10.1257/pandp.20251112}
+}
+
+@article{GowrisankaranJpe2025,
+author = {Gowrisankaran, Gautam and Langer, Ashley and Zhang, Wendan},
+title = {Policy Uncertainty in the Market for Coal Electricity: The Case of Air Toxics Standards},
+journal = {Journal of Political Economy},
+volume = {133},
+number = {6},
+pages = {1757-1795},
+year = {2025},
+doi = {10.1086/734779},
+URL = {https://doi.org/10.1086/734779},
+eprint = {https://doi.org/10.1086/734779},
```

Comment:
The eprint field seems to be generating a bad link:

[arXiv:https://doi.org/10.1086/734779](https://arxiv.org/abs/https://doi.org/10.1086/734779)

And actually ctrl-F-ing through the bibliography I see that there are a lot of other bad arXiv links too (and a few good ones)